source/modules/MEI.cmn.xml

```diff
@@ -1346,10 +1346,10 @@
     </content>
     <constraintSpec ident="When_not_copyof_beam_content" scheme="isoschematron">
       <constraint>
-        <sch:rule context="mei:beam[not(@copyof)]">
+        <sch:rule context="mei:beam[not(@copyof) and not(@sameas)]">
```

Comment:
is it "AND" or "OR" ?

Reply:
SInce the text says "or"

Reply:
true AND true = true; 
true AND false = false; (rule does not apply when one attribute is not there but the other is)
true AND true = false;

ture OR true = true;
true OR false = true (rule applies when only one attribute is not there)
false OR false = false

Or do I mess up and and and and and and or?

Reply:
Wow. You win the gold star for that last sentence. Perfect. ⭐

Reply:
Good catch.  I think it's "or".

Reply:
No, it definitely has to be `and` because what we want to express in words is:

If a beam has neither `@copyof` nor `@sameas` it has to have at least two descendants.

With `or` the expression would evaluate true if just one of them was missing.